packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-card.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+import { isMobile } from '@deriv/shared';
+import { AppLinkedWithWalletIcon } from '../app-linked-with-wallet-icon';
+import { TWalletCFDCard } from './wallet-cfd-success-dialog';
+import { localize } from '@deriv/translations';
```

Comment:
@aizad-deriv I'm not sure about using `localize` in `components` package. As I know we can't use it here